lib/hako/schedulers/ecs_elb.rb

```diff
@@ -61,6 +61,25 @@ def find_or_create_load_balancer(front_port)
         true
       end
 
+      # @return [Boolean]
+      def modify_attributes
+        attributes = {}
+        if @elb_config.key?('cross_zone_load_balancing')
+          attributes.merge!(
+            cross_zone_load_balancing: {
+              enabled: @elb_config['cross_zone_load_balancing'],
+            }
+          )
+        end
+        unless attributes.empty?
+          @elb.modify_load_balancer_attributes(
+            load_balancer_name: name,
+            load_balancer_attributes: attributes,
+          )
+        end
+        !attributes.empty?
```

Comment:
What is the purpose of the return value? It looks it doesn't make any sense to me.